app/repository/RepositoryImplementation.java

```diff
@@ -12,11 +12,6 @@
 public class RepositoryImplementation<T> implements Repository<T> {
 
     @Override
-    public T findById(Long id) {
```

Comment:
This should belong here, revert this removed code.
In implementation of this method here, you should call `JPA.em().find` method.